lambda_function.py

```diff
@@ -47,20 +55,24 @@ def lambda_handler(event, context, debug=False):
 
     base_repo_owner = message['pull_request']['base']['repo']['owner']['login']
     base_repo = message['pull_request']['base']['repo']['name']
+    base_repo_full_name = message['pull_request']['base']['repo']['full_name']
+
     base_branch = message['pull_request']['base']['ref']
     head_branch = message['pull_request']['head']['ref']
 
-    if base_repo_owner != config.repo_owner or base_repo != config.repo:
-        print("Got event for unexpected repo {}/{}".format(
-            base_repo_owner, base_repo))
+    if base_repo_full_name not in config.repos:
+        print("Got event for unexpected repo {}".format(base_repo_full_name))
         return
 
-    if base_branch in config.ignore_base_branch:
+    repo_config = BASE_REPO_CONFIG.copy()
+    repo_config.update(config.repos[base_repo_full_name])
```

Comment:
This approach is pretty okay. I'm down to use it like this without ChainMap. Just make sure you also add a default one.